src/handlebars.l

```diff
@@ -49,12 +49,18 @@ ID    [^\s!"#%-,\.\/;->@\[-\^`\{-~]+/{LOOKAHEAD}
                                    return 'CONTENT';
                                  }
 
+// nested raw block will create stacked 'raw' condition
+<raw>"{{{{"/[^/]                 this.begin('raw'); return 'CONTENT';
 <raw>"{{{{/"[^\s!"#%-,\.\/;->@\[-\^`\{-~]+/[=}\s\/.]"}}}}" {
-                                  yytext = yytext.substr(5, yyleng-9);
                                   this.popState();
-                                  return 'END_RAW_BLOCK';
+                                  if (this.conditionStack[this.conditionStack.length-1] === 'raw') {
```

Comment:
Should be using `this.topState()` here, but it currently returns the second top instead of the first top. Opened an issue about it at https://github.com/zaach/jison/issues/291